x/assets/types/general.go

```diff
@@ -11,6 +11,15 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 )
 
+const (
+	NativeChainLzID  = 0
+	NativeAssetAddr  = "0x0000000000000000000000000000000000000000"
+	NativeAssetID    = "0x0000000000000000000000000000000000000000_0x0"
+	NativeAssetDenom = "aexo"
```

Comment:
Use `utils.BaseDenom` with `"github.com/evmos/evmos/v14/utils"`

Reply:
Should it be "github.com/ExocoreNetwork/exocore/utils" ? updated with `eoxore/utils`